.github/workflows/publish-docker.yaml

```diff
@@ -61,7 +61,7 @@ jobs:
         env:
           MAVEN_OPTS: -Xmx2G -Xms2G
         run: |
-          ./mvnw -B clean deploy \
+          ./mvnw -T 16 -B clean deploy \
```

Comment:
`-T` is an unsafe option, I think we shouldn't use it in deploy command.

Reply:
Sure, let me remove it

Reply:
@leo65535 Thanks a lot for your comment. However, I didn't fully understand what's your concern. Would you please explain more details. Anyway, I have removed this in case something goes wrong.